models/address_book.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'entry'
+
+class AddressBook
+  attr_accessor :entries
+
+  def initialize
+    @entries = []
+  end
+
+  def add_entry(name, phone_number, email)
+    index = 0
+    @entries.each do |entry|
+      if name < entry.name
+        break
+      end
+      index += 1
+    end
+
+    @entries.insert(index, Entry.new(name, phone_number, email))
+  end
+# Code for assignment: remove_entry method.
+  def remove_entry(name, phone_number, email)
+    delete_entryj = nil
+
+    @entries.each do |entry|
+      if name == entry.name && phone_number == entry.phone_number && email == entry.email
+        @delete_entry = entry
```

Comment:
Creating an instance variable means it will be available outside of the the method (remove_entry, in this case).